pkg/api/v1beta3/types.go

```diff
@@ -1691,7 +1691,7 @@ type ComponentCondition struct {
 // ComponentStatus (and ComponentStatusList) holds the cluster validation info.
 type ComponentStatus struct {
 	TypeMeta   `json:",inline"`
-	ObjectMeta `json:"metadata,omitempty"`
+	ObjectMeta `json:"metadata,omitempty" description:"standard object metadata; see https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api-conventions.md#metadata"`
 
 	Name       string               `json:"name,omitempty" description:"name of the component"`
 	Conditions []ComponentCondition `json:"conditions,omitempty" description:"Conditions is a list of ComponentCondition objects"`
```

Comment:
I'll change the description. But for my education, what's patchStrategy and patchMergeKey? If these are for writes, then they may not be needed for ComponenStatus, since it's a read-only object for now.